getting-started/kubearmor_visibility.md

```diff
@@ -0,0 +1,128 @@
+# KubeArmor Visibility
+
+KubeArmor currently supports enabling visibility for containers/hosts
+Visibility for containers is enabled by default and can be accesed using `karmor` tool
+
+
+
+## Getting Container Visibility
+### Example : multiubuntu
+* Prerequisites
+
+ The following  example would require a K8s cluster , please follow [this](https://github.com/kubearmor/KubeArmor/blob/main/getting-started/deployment_guide.md#prerequisites) to setup one if you dont have access to one
+
+
+* Install karmor cli tool.
+
+  ```text
+  $ curl -sfL http://get.kubearmor.io/ | sudo sh -s -- -b /usr/local/bin
+  ```
+* Install KubeArmor
+
+  ```text
+  $ karmor install
+  ```
```

Comment:
We do not have to repeat the installation steps here. We can point to the getting started guide. The problem is if there are any changes to the getting started guide then multiple pages have to be updated.